### PR TITLE
优化在 Windows 下文章编辑页字体

### PR DESCRIPTION
**Pull request by Vndroid**
- 格式化CSS
- 优化 Windows / macOS 下的字体，视窗默认使用「微软雅黑」

### CODE REVIEW

**Comment by scvoet** [comment]
感觉没有必要进行格式化。现在的方案不至于让 css 和压缩后的一样无法阅读，也在一定程度上进行了压缩。

**Comment by Vndroid** [comment]
> 感觉没有必要进行格式化。现在的方案不至于让 css 和压缩后的一样无法阅读，也在一定程度上进行了压缩。

感觉没有必要回复我。建议你好好看看更新的内容，搞清楚重点。

File: admin/css/grid.css [hunk 1]
```diff
@@ -1,211 +1,744 @@
...
-  .col-push-4 { left: 33.33333%; }
-  .col-push-5 { left: 41.66667%; }
-  .col-push-6 { left: 50%; }
-  .col-push-7 { left: 58.33333%; }
-  .col-push-8 { left: 66.66667%; }
-  .col-push-9 { left: 75%; }
-  .col-push-10 { left: 83.33333%; }
-  .col-push-11 { left: 91.66667%; }
-  .col-push-12 { left: 100%; } }
+@media (min-width: 992px) {
```
Comment (KevinZonda) [inline admin/css/grid.css:292]: I don't think codes of css animation should be expanded.

File: admin/css/grid.css [hunk 2]
```diff
@@ -1,211 +1,744 @@
...
+}
 
-.row [class*="-push-"], .row [class*="-pull-"] { position: relative; }
+.row [class*="-push-"], .row [class*="-pull-"] {
+    position: relative;
+}
 
 /* Mobile and up */
-.col-mb-1 { width: 8.33333%; }
+.col-mb-1 {
```
Comment (KevinZonda) [inline admin/css/grid.css:32]: No need to expand.

File: admin/css/grid.css [hunk 3]
```diff
@@ -1,211 +1,744 @@
...
-  .col-tb-push-4 { left: 33.33333%; }
-  .col-tb-push-5 { left: 41.66667%; }
-  .col-tb-push-6 { left: 50%; }
-  .col-tb-push-7 { left: 58.33333%; }
-  .col-tb-push-8 { left: 66.66667%; }
-  .col-tb-push-9 { left: 75%; }
-  .col-tb-push-10 { left: 83.33333%; }
-  .col-tb-push-11 { left: 91.66667%; }
-  .col-tb-push-12 { left: 100%; } }
+@media (min-width: 768px) {
```
Comment (KevinZonda) [inline admin/css/grid.css:81]: I don't think codes of css animation should be expanded.

File: admin/css/grid.css [hunk 4]
```diff
@@ -1,211 +1,744 @@
...
-  .col-push-5 { left: 41.66667%; }
-  .col-push-6 { left: 50%; }
-  .col-push-7 { left: 58.33333%; }
-  .col-push-8 { left: 66.66667%; }
-  .col-push-9 { left: 75%; }
-  .col-push-10 { left: 83.33333%; }
-  .col-push-11 { left: 91.66667%; }
-  .col-push-12 { left: 100%; } }
+@media (min-width: 992px) {
+    .container {
```
Comment (KevinZonda) [inline admin/css/grid.css:293]: No need to expand.

File: admin/css/style.css [hunk 5]
```diff
@@ -2,933 +2,2532 @@
...
+input.text-s, textarea.text-s {
+    padding: 5px;
+}
+
+input.text-l, textarea.text-l {
+    padding: 10px;
+    font-size: 1.14286em;
+}
+
+.w-10 {
```
Comment (KevinZonda) [inline admin/css/style.css:91]: No need to expand.

File: admin/css/style.css [hunk 6]
```diff
@@ -2,933 +2,2532 @@
...
+}
+
+.typecho-page-main ul.tag-list li {
+    display: inline-block;
+    margin: 0 0 5px 0;
+    padding: 5px 5px 5px 10px;
+    cursor: pointer;
+}
+
+.typecho-page-main ul.tag-list li:hover {
```
Comment (KevinZonda) [inline admin/css/style.css:1202]: No need to expand.

File: admin/css/style.css [hunk 7]
```diff
@@ -2,933 +2,2532 @@
...
+    width: 16px;
+    height: 16px;
+}
+
+.i-upload, .i-upload-active {
+    width: 24px;
+    height: 24px;
+}
+
+.i-edit {
```
Comment (KevinZonda) [inline admin/css/style.css:1586]: No need to expand.

File: admin/css/style.css [hunk 8]
```diff
@@ -2,933 +2,2532 @@
...
+    .editor-sprite, .editor-editor-bold, .editor-editor-code, .editor-editor-exit-fullscreen, .editor-editor-fullscreen, .editor-editor-heading, .editor-editor-hr, .editor-editor-image, .editor-editor-italic, .editor-editor-link, .editor-editor-more, .editor-editor-olist, .editor-editor-quote, .editor-editor-redo, .editor-editor-ulist, .editor-editor-undo, .wmd-button-row li#wmd-bold-button span, .wmd-button-row li#wmd-italic-button span, .wmd-button-row li#wmd-link-button span, .wmd-button-row li#wmd-quote-button span, .wmd-button-row li#wmd-code-button span, .wmd-button-row li#wmd-image-button span, .wmd-button-row li#wmd-olist-button span, .wmd-button-row li#wmd-ulist-button span, .wmd-button-row li#wmd-heading-button span, .wmd-button-row li#wmd-hr-button span, .wmd-button-row li#wmd-more-button span, .wmd-button-row li#wmd-undo-button span, .wmd-button-row li#wmd-redo-button span, .wmd-button-row li#wmd-fullscreen-button span, .wmd-button-row li#wmd-exit-fullscreen-button span {
+        background-image: url("../img/editor@2x.png?_=f3643ed");
+        background-repeat: no-repeat;
+        background-size: 20px 300px;
+    }
+
+    .editor-editor-bold {
```
Comment (KevinZonda) [inline admin/css/style.css:1801]: No need to expand.

File: admin/css/style.css [hunk 9]
```diff
@@ -2,933 +2,2532 @@
...
+    cursor: default;
+}
+
+.wmd-button-row li span {
+    display: block;
+    width: 20px;
+    height: 20px;
+}
+
+.wmd-button-row li#wmd-bold-button span {
```
Comment (KevinZonda) [inline admin/css/style.css:1958]: No need to expand.

**Comment by KevinZonda** [comment]
@Vndroid If u just use an automatic tool to format, I think this PR shouldn't be merge. BTW, U destroy the whole structure of the css files that will also rocket the difficulty of development.

**Pull request closed without merging**